Resources/Prototypes/_CP14/Entities/Structures/Storage/Crates/coffin.yml

```diff
@@ -0,0 +1,33 @@
+- type: entity
+  parent: CrateGeneric
+  id: CP14CrateCoffin
+  name: гроб
+  description: Удобный и красивый гроб, чтобы с удобствами дождаться своего воскрешения.
```

Comment:
need eng name and descruption + ru locale in Locale/ru-RU/_CP14/_PROTO

Reply:
ready